_docs-sources/guides/style/golang-style-guide.md

```diff
@@ -181,6 +181,20 @@ Since many languages use `ALL_CAPS` for constants, it is worth calling out expli
 Therefore, `region` or `testRegion` for private constants and `Region` or `TestRegion` for public ones is preferred over
 `REGION` or `TEST_REGION`.
 
+
+### Using packages
+
+A common Go pattern is creating new packages under the `pkg` or `internal` directories. If you are unsure about which
+one to use, follow this simple litmus test:
+
+- The `internal` directory contains code that we don't want to be consumed by other clients.
+- Telemetry is a great example, as it is typically app-specific and may have a private DSN.
```

Comment:
Nit: indent this bullet point one level under the one on line 190.

Reply:
fixed.